packages/imperative/src/imperative/__tests__/plugins/cmd/install/install.handler.unit.test.ts

```diff
@@ -379,5 +381,90 @@ describe("Plugin Management Facility install handler", () => {
         expect(expectedError.additionalDetails).toContain("Command failed");
         expect(expectedError.additionalDetails).toContain("npm");
     });
+    it("should handle installed plugins via public scope", async () => {
+        const handler = new InstallHandler();
+
+        const params = getIHandlerParametersObject();
+        params.arguments.plugin = ["@public/sample1"];
+
+        mocks.getScopeRegistry.mockReturnValueOnce("publicRegistryUrl" as any);
+
+        try {
+            await handler.process(params);
+        } catch (e) {
+            expect(e).toBeUndefined();
+        }
+
+        expect(mocks.install).toHaveBeenCalledWith("@public/sample1","publicRegistryUrl");
+    });
+    it("should handle installed plugins via private scope", async () => {
+        const handler = new InstallHandler();
+
+        const params = getIHandlerParametersObject();
+        params.arguments.plugin = ["@private/sample1"];
+
+        mocks.getScopeRegistry.mockReturnValueOnce("privateRegistryUrl" as any);
+
+        try{
+            await handler.process(params);
+        }
+        catch(e){
+            expect(e).toBeUndefined();
+        }
+
+        expect(mocks.install).toHaveBeenCalledWith("@private/sample1","privateRegistryUrl");
+    });
```

Comment:
This test seems redundant with the one above for public scope, unless the install handler takes a different code path based on whether the scope is public or private.

Reply:
resolved